### PR TITLE
add choice after sweet marshmallow

### PR DESCRIPTION
**Pull request by michaelpenrose**

### CODE REVIEW

**Comment by michaelpenrose** [comment]
Please review this, @casey-collab

File: english/sing/lightup/accept/reciprocate/cake.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+The monster is very happy with the cake you offer him.
```
Comment (casey-collab) [inline english/sing/lightup/accept/reciprocate/cake.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/sing/lightup/accept/reciprocate/water.md [hunk 2]
```diff
@@ -0,0 +1 @@
+The marshmallow monster gratefully accepts and says "Wow I needed that - I was so thirsty."
```
Comment (casey-collab) [inline english/sing/lightup/accept/reciprocate/water.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by michaelpenrose** [comment]
OK @casey-collab I have made the changes to put cake.md and water.md each into their own folders. Please review again.

**Comment by casey-collab** [comment]
Unfortunately this pull request could not be automatically reviewed due to technical difficulty.

**Comment by michaelpenrose** [comment]
can we try one more time @casey-collab ?

File: english/sing/lightup/accept/accept.md [hunk 3]
```diff
@@ -1 +1,11 @@
...
+
+You are happy and decide to offer something back to your new friend.
+
+Do you:
+
+[Offer him a drink of water?](reciprocate/water.md)
+
+or
+
+[Offer him a piece of cake?](reciprocate/cake.md)
```
Comment (casey-collab) [inline english/sing/lightup/accept/accept.md]: This link is also invalid.

File: english/sing/lightup/accept/cake/cake.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+The monster is very happy with the cake you offer him.
```
Comment (casey-collab) [inline english/sing/lightup/accept/cake/cake.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/sing/lightup/accept/water/water.md [hunk 5]
```diff
@@ -0,0 +1 @@
+The marshmallow monster gratefully accepts and says "Wow I needed that - I was so thirsty."
```
Comment (casey-collab) [inline english/sing/lightup/accept/water/water.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/sing/lightup/accept/accept.md [hunk 6]
```diff
@@ -1 +1,11 @@
-The marshamallow treats are delicious!
\ No newline at end of file
+The marshamallow treats are delicious!
+
+You are happy and decide to offer something back to your new friend.
+
+Do you:
+
+[Offer him a drink of water?](reciprocate/water.md)
```
[the remark this answers is not in the document]
Comment (michaelpenrose) [reply english/sing/lightup/accept/accept.md]: ok you were right. I fixed that in the latest version.

File: english/sing/lightup/accept/cake/cake.md (hunk 4 above)
Comment (michaelpenrose) [reply english/sing/lightup/accept/cake/cake.md]: ok you were right. I fixed that in the latest version.

**Comment by michaelpenrose** [comment]
can we try again @casey-collab ?

**Comment by casey-collab** [comment]
Looks great! :thumbsup:

**Pull request merged**